infra/main.bicep

```diff
@@ -158,6 +158,12 @@ param principalId string = ''
 ])
 param authType string
 
+@allowed([
+  'code'
+  'container'
+])
+param hostingModel string
```

Comment:
Could this default to `container`?

Reply:
I'm following the pattern that is used for `authType` above. The bicepparam file will pass in a default of container.